src/parser.ts

```diff
@@ -265,6 +270,8 @@ function createStatementParserByToken(
         return createSelectStatementParser(options);
       case 'CREATE':
         return createCreateStatementParser(options);
+      case 'SHOW':
+        return createShowStatementParser(options);
```

Comment:
Since you say that these are only available for MySQL dialect, should then we wrap this call with:

```suggestion
        if (options.dialect === 'mysql') {
          return createShowStatementParser(options);
        }
        break;
```

Reply:
The thought was to keep it open just in case other dialects ended up using show like this, I just wanted to specify that it was being used by a mysql implementation and from what I could see was only mysql specific. I can wrap it to if dialect is mysql OR generic if you'd like. Let me know.

Reply:
@MasterOdin let me know how you want me to proceed with this.

Reply:
I'm good with the wrap that you've added that supports both mysql + generic. I'd ask that you also update the readme to indicate that generic will also work for them.